rl4co/envs/graph/mcp/render.py

```diff
@@ -0,0 +1,14 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+
+from rl4co.utils.ops import gather_by_index
+from rl4co.utils.pylogger import get_pylogger
+
+log = get_pylogger(__name__)
+
+
+def render(td, actions=None, ax=None):
+    # TODO: better rendering, e.g., visualization 
```

Comment:
Something missing here I guess? If not implemented, we can simply do `raise NotImplementedError`

Reply:
Thanks. I simply removed it for now.